File: aisile-sellergoods-interface/src/main/java/com/aisile/sellergoods/service/SellerService.java
package com.aisile.sellergoods.service;

import java.util.List;

import com.aisile.pojo.TbSeller;
import com.aisile.pojo.entity.PageResult;
import com.aisile.pojo.entity.Result;

public interface SellerService {
	
	public List<TbSeller> findAll();
	
	public PageResult findPage( int page,int rows);
	
	public PageResult findSearch(int page,int rows,TbSeller tbSeller);
	
	public void add(TbSeller tbSeller );
	
	public void update(TbSeller tbSeller);
	
	public TbSeller findOne(String id); 
	
	public Result delete(Long [] ids);

	public void updateStatus(TbSeller tbSeller, String status);
}

File: aisile-manager-web/src/main/webapp/js/service/sellerService.js
//封装所有连接后端代码    需要$http
app.service('sellerService',function($http){
	//查询全部的数据
	this.findAll = function(){
		return $http.get('../seller/findAll.do');
	}
	//分页
	this.findPage = function(page,rows){
		return $http.get('../seller/findPage.do?page='+page+"&rows="+rows);
	}
	//分页带数据查询
	this.findSearch = function(page,rows,searchEntity){
		return $http.post('../seller/findSearch.do?page='+page+"&rows="+rows,searchEntity);
	}
	//修改
	this.update = function(entity){
		return $http.post('../seller/update.do',entity);
	}
	//修改(针对审核使用)
	this.updateStatus = function(entity,status){
		return $http.post('../seller/updateStatus.do?status='+status,entity);
	}
	//添加
	this.add = function(entity){
		return $http.post('../seller/add.do',entity);
	}
	//根据id查询对象
	this.findOne = function(id){
		return $http.post('../seller/findOne.do?id='+id);
	}
	//删除
	this.dele = function(selectIds){
		return $http.get('../seller/dele.do?ids='+selectIds);
	}
	//下拉框
	this.selectOptionList = function(){
		return $http.get('../seller/selectOptionList.do');
	}
})